client_test.go

```diff
@@ -252,7 +252,7 @@ func TestConcurrency(t *testing.T) {
 		defer close(errors)
 
 		for i := 0; i <= iterations; i++ {
-			if _, err := c.Server.GroupList(); err != nil {
+			if _, err := c.Server.GroupList(); err != nil { // nolint: vetshadow
```

Comment:
This is never correct, this should be:
```golang
if _, err = c.Server.GroupList(); err != nil {
```

Reply:
I wouldn't judge too quick, this is what I had before the fore-push and didn't work because it causes a data-race (see [here](https://travis-ci.org/irgendwr/go-ts3/jobs/491113869#L530)) so unless I rename `err` to something different shadowing is indeed correct 😄

Reply:
Ahh yes then following with prevent the shadowing warning:
```golang
if _, err2 := c.Server.GroupList(); err2 != nil {
...
```